src/containers/NetworkViewers/RelationshipTreeViewer.tsx

```diff
@@ -150,12 +343,86 @@ class TreeViewer extends Component<Props, State> {
     };
   };
 
+  renderLegend = () => {
+    const nodes: { [key: string]: RelationshipResource } = {
+      'Current Asset': {
+        resource: 'asset',
+        resourceId: `${this.props.app.assetId}`,
+      },
+      Asset: { resource: 'asset', resourceId: '-1' },
+      Timeseries: { resource: 'timeSeries', resourceId: '-1' },
+      '3D': { resource: 'threeD', resourceId: '-1:-1' },
+    };
+    const relationships: { [key: string]: Relationship } = {
+      'Flows To': {
+        relationshipType: 'flowsTo',
+        source: nodes.Asset,
+        target: nodes.Asset,
+        confidence: 1,
```

Comment:
Why hard coded to 1?

Reply:
because in this case it doesnt matter :P just for generating the legend cleanly